### PR TITLE
CSGOROADTOGLOBAL

### PR DESCRIPTION
**Pull request by Roytak**

### CODE REVIEW

File: frontroomsbot/bot.py [hunk 1]
```diff
@@ -54,6 +55,26 @@ async def sync(interaction: discord.Interaction):
     await interaction.response.send_message("Synced!")
     print("Command tree synced")
 
+@tree.command(name="csgoroadtoglobal", description="CSGOROADTOGLOBAL", guild=guild)
+async def csgo(interaction: discord.Interaction):
+    # list of zmrds
+    zmrdi = ['JosefKuchar', 'Roytak', 'Soromis', 'TominoBLM', 'Addy Daddy', 'Kubik', 'MrStinky']
```
Comment (TominoFTW) [inline frontroomsbot/bot.py]: Taguj pres ID, ne pres nick :skull:

Comment (S1ro1) [inline frontroomsbot/bot.py]: I wish to be the person I was before seeing this

**Review by S1ro1 — CHANGES_REQUESTED** [review]
Pls fix...everything

File: frontroomsbot/bot.py (hunk 1 above)
Comment (Roytak) [reply frontroomsbot/bot.py]: I fixed!! It's ok now I think, but please check, master.

**Review by Roytak — COMMENTED** [review]
LGTM

**Pull request closed without merging**